moz_kinto_publisher/main.py

```diff
@@ -1248,71 +1235,46 @@ def publish_crlite(*, args, ro_client, rw_client):
     existing_records = ro_client.get_records(
         collection=settings.KINTO_CRLITE_COLLECTION
     )
+    # Sort existing_records for crlite_verify_record_sanity
```

Comment:
Might be helpful to add something like ", which gets called in `crlite_determine_publish`"

Reply:
OK